src/sql/base/browser/ui/infoBox/infoBox.ts

```diff
@@ -43,10 +54,30 @@ export class InfoBox extends Disposable implements IThemable {
 		container.appendChild(this._infoBoxElement);
 		this._infoBoxElement.appendChild(this._imageElement);
 		this._infoBoxElement.appendChild(this._textElement);
+		this._clickableIndicator = DOM.$('a');
+		this._clickableIndicator.classList.add('infobox-clickable-arrow', ...Codicon.arrowRight.classNamesArray);
+		this._infoBoxElement.appendChild(this._clickableIndicator);
+
+		this._register(DOM.addDisposableListener(this._infoBoxElement, DOM.EventType.CLICK, e => {
```

Comment:
You should only register these if it's clickable - and unregister it if it's changed to not be clickable anymore. Otherwise it'll always be announced as "clickable" by screen readers.

Reply:
These shouldn't be here - you should only set it when isClickable is set to true